robottelo/manifests.py

```diff
@@ -103,10 +105,14 @@ def original(self):
             self._download_manifest_info()
         return six.BytesIO(self.template)
 
-
+if not settings.configured:
```

Comment:
Missed this, but avoid calling settings.configure at import time. Configuring at import time will break documentation generation.